### ISSUE TITLE
[Bug]: Invalid regular expression: invalid group specifier name (introduced in 2.10.3)

### ISSUE THREAD

**Issue by erwamartin** [ueberdosis/tiptap#5889]
### Affected Packages

extension-code

### Version(s)

2.10.3

### Bug Description

The TipTap code extension fails to load on Safari versions below 16 due to the following error:

Invalid regular expression: invalid group specifier name.

This issue prevents the file from being loaded and stems from an incompatible regex introduced in this commit:
ueberdosis/tiptap commit f79c05e.

The regex uses named capturing groups and lookbehind assertions, which are not supported in Safari versions earlier than 16.

### Browser Used

Safari

### Expected Behavior

The file should load correctly.

**Rewritten Regex**
_Original:_
```
export const inputRegex = /(?<!`)`([^`]+)`(?!`)/;
export const pasteRegex = /(?<!`)`([^`]+)`(?!`)/g;
```

_Rewritten Without Lookbehind:_
```
export const inputRegex = /(^|[^`])`([^`]+)`(?!`)/;
export const pasteRegex = /(^|[^`])`([^`]+)`(?!`)/g;
```

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

**Comment by wallisonfelipe** [comment]
Same here:
Invalid regular expression: invalid group specifier name

**Comment by nperez0111** [comment]
Would happily accept a PR to address this

**Comment by lino-levan** [comment]
This issue was fixed in #5916. I guess we're waiting on that to be merged.

**Comment by nperez0111** [comment]
Yes, we queue releases. If you need it now, you cna use the PR's version: https://github.com/ueberdosis/tiptap/pull/5916#issuecomment-2519498744

**Comment by lino-levan** [comment]
Any timeline on when the next version of tiptap will be released?

**Comment by nperez0111** [comment]
No, we release when we feel ready. Please refer to my previous comment above
> If you need it now, you can use the PR's version

**Comment by ccreusat** [comment]
Hi guys !

For your information, we have the same problem with Tiptap and the code extension in version `2.3.0` in Safari 15.6 and below (but that's ok I think). Works in Safari 16.5, 17.6, ... 👍🏼

**Comment by Nantris** [comment]
Whoops I see I've duplicated this issue in #5948.

I've closed it as a duplicated, but I've suggested a more robust approach there that brings the best of both worlds.

**Comment by Nantris** [comment]
I think this was already released though and issues persist? So now I'm not sure I've really duplicated the issue. Anyway, sorry, that's enough cluttering your inboxes. Looking forward to a resolution.

**Comment by Nantris** [comment]
Friendly bump. Not sure if this is a duplicate of #5948 or not, but it's keeping us stuck on `2.10.0` which is now a number of versions behind the latest.

**Comment by nperez0111** [comment]
This was released yesterday already with 2.11.0
